sdk/include/ndk/setypes.h

```diff
@@ -29,6 +29,16 @@ Header Name:
 //
 #define SECURITY_INTERNETSITE_AUTHORITY     {0,0,0,0,0,7}
 
+//
+// Privilege token filtering flags
+//
+#define DISABLE_MAX_PRIVILEGE 0x1
+#define SANDBOX_INERT         0x2
+#if (NTDDI_VERSION >= NTDDI_LONGHORN)
+#define LUA_TOKEN             0x4
+#define WRITE_RESTRICTED      0x8
+#endif
```

Comment:
Surprising this needs to be in the NDK (as they appear to be documented... at least in MSDN). Do you know whether they are in some public header in the Win2003 PSDK?

Reply:
Seemingly these appear in `winnt.h`. Thomas told me that for this case these have to be in NDK, not XDK.

Reply:
To clarify: they should probably be in both.
* Somewhere in XDK such that they end up in `winnt.h` for UM code
* In NDK for KM code, since other KM headers don't define them and we need them in ntoskrnl